docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+db:
+  image: mysql
+  volumes:
+    - ~/.docker-volumes/openproject/db:/var/lib/mysql
+  expose:
+    - "3306"
+  environment:
+    - MYSQL_ALLOW_EMPTY_PASSWORD=true
+
+worker: &ruby
+  build: .
+  volumes:
+    - .:/usr/src/app
+  links:
+    - db
+  environment:
+    - "DATABASE_URL=mysql2://root@db/openproject_development?encoding=utf8"
```

Comment:
True, but I think it makes sense to avoid any additional steps to get a dev env running. Also, you can manually set another `DATABASE_URL` before launching tests in the container.